src/main/resources/google_checks.xml

```diff
@@ -91,6 +91,7 @@
         <module name="SeparatorWrap">
             <property name="id" value="SeparatorWrapComma"/>
             <property name="tokens" value="COMMA"/>
+            <property name="tokens" value="METHOD_REF"/>
```

Comment:
We need confirmation that the google style wants EOL for this token.
Please look at http://checkstyle.sourceforge.net/reports/google-java-style-20160712.html and make note if it references method references and how they should wrap. You may have to read between the lines with google's style guide.

If it does make reference, point me to the area, if not, I will also confirm, and we will most likely have to remove this from the check and make an override to give a reason why it isn't added.